WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsListAdapter.kt

```diff
@@ -0,0 +1,77 @@
+package org.wordpress.android.ui.bloggingprompts
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.view.ViewGroup.LayoutParams
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.DiffUtil.Callback
+import androidx.recyclerview.widget.RecyclerView
+import org.wordpress.android.R
+import org.wordpress.android.databinding.BloggingPromptsListItemBinding
+import org.wordpress.android.util.extensions.setVisible
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class BloggingPromptsListAdapter() : RecyclerView.Adapter<BloggingPromptsListItemViewHolder>() {
+    private var items = listOf<BloggingPromptsListItem>()
+
+    fun update(newItems: List<BloggingPromptsListItem>) {
+        val diffResult = DiffUtil.calculateDiff(
+                BloggingPromptsListDiffCallback(
+                        items,
+                        newItems
+                )
+        )
+        items = newItems
+        diffResult.dispatchUpdatesTo(this)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): BloggingPromptsListItemViewHolder {
+        val layoutInflater = LayoutInflater.from(parent.context)
+        val binding = BloggingPromptsListItemBinding.inflate(layoutInflater, parent, false)
+        return BloggingPromptsListItemViewHolder(binding)
+    }
+
+    override fun getItemCount() = items.size
+
+    override fun onBindViewHolder(holder: BloggingPromptsListItemViewHolder, position: Int) {
+        holder.bind(items[position])
+    }
+}
+
+class BloggingPromptsListItemViewHolder(
+    private val binding: BloggingPromptsListItemBinding
+) : RecyclerView.ViewHolder(binding.root) {
+    fun bind(prompt: BloggingPromptsListItem) {
+        with(binding) {
+            promptTitle.text = prompt.title
+            promptSubtitleAnswerCount.text =
+                    when (prompt.respondentsCount) {
```

Comment:
Since we can't use plurals in the project, do you think it would make sense to extract this into an extension function, so it can be reused? Something like:

```
    fun View.getQuantityString(
        quantity: Int,
        @StringRes zeroRes: Int,
        @StringRes oneRes: Int,
        @StringRes otherRes: Int
    ) = when (quantity) {
        0 -> context.getString(zeroRes)
        1 -> context.getString(oneRes)
        else -> context.getString(otherRes)
    }
```

Reply:
Makes sense. Will make it an extension method and move it to `ViewExtensions.kt`

Reply:
This is done.